rst_files/julia_by_example.rst

```diff
@@ -49,9 +47,9 @@ In this lecture, we will often start with a direct MATLAB/FORTRAN approach which
 Set Up
 --------
 
-We assume that you've worked your way through :doc:`our getting started lecture <getting_started>` already
+We assume that you've worked your way through [our getting started lecture](getting_started.ipynb) already
 
-In particular, the easiest way to install and precompile all of the Julia packages used in the QuantEcon notes is to go ``] add InstantiateFromURL`` and then work in a Jupyter notebook, as described :ref:`here <jl_jupyter>`
+In particular, the easiest way to install and precompile all of the Julia packages used in the QuantEcon notes is to go ``] add InstantiateFromURL`` and then work in a Jupyter notebook, as described [here](getting_started.ipynb)
```

Comment:
I think this is markdown rather than RST links

Reply:
```suggestion
In particular, the easiest way to install and precompile all of the Julia packages used in the QuantEcon notes is to go ``] add InstantiateFromURL`` and then work in a Jupyter notebook, as described ` here <getting_started>`_
```